### PR TITLE
fix: kaspersky filter pattern overmatching sdata

### PR DESCRIPTION
**Pull request by ikheifets-splunk**

### CODE REVIEW

File: tests/test_vmware.py [hunk 1]
```diff
@@ -500,4 +500,40 @@ def test_linux_vmware_usbarb(record_property, setup_wordlist, setup_splunk, setu
     record_property("resultCount", resultCount)
     record_property("message", message)
 
+    assert resultCount == 1
+
+def test_vmware_overlapping_with_another_sdata(
```
Comment (ikheifets-splunk) [inline tests/test_vmware.py:505]: Originally in issue `kasperksy` filter overmatch value of `VMware`

File: package/etc/conf.d/conflib/syslog/app-syslog-kaspersky_syslog.conf [hunk 2]
```diff
@@ -25,7 +25,7 @@ block parser app-syslog-kaspersky_syslog() {
 };
 application app-syslog-kaspersky_syslog[sc4s-syslog-sdata] {
 	filter {
-        match('@23668' value('SDATA'))
+        match('[[^\@]@23668' value('SDATA'))
```
Comment (ikheifets-splunk) [inline package/etc/conf.d/conflib/syslog/app-syslog-kaspersky_syslog.conf:28]: also might use `[event@23668`, but it's seems that would be more generic

File: package/etc/conf.d/conflib/syslog/app-syslog-kaspersky_syslog.conf [hunk 3]
```diff
@@ -25,8 +25,7 @@ block parser app-syslog-kaspersky_syslog() {
 };
 application app-syslog-kaspersky_syslog[sc4s-syslog-sdata] {
 	filter {
-        match('@23668' value('SDATA'))
-
+        match('event@23668' value('SDATA'))
```
Comment (rjha-splunk) [inline package/etc/conf.d/conflib/syslog/app-syslog-kaspersky_syslog.conf:28]: filter {                                                        
        match('@23668' value('.SDATA.*'))                   
    };

The filter should be updated as there is still a chance in main data there is event@23668888 etc.

**Review by rjha-splunk — CHANGES_REQUESTED** [review]
https://github.com/splunk/splunk-connect-for-syslog/pull/2126/files#r1281776830 should be updated

**Review by rjha-splunk — APPROVED** [review]

**Pull request merged**